src/WebJobs.Script.WebHost/WebHooks/WebHookReceiverManager.cs

```diff
@@ -116,15 +116,8 @@ protected virtual void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                    if (_httpConfiguration != null)
-                    {
-                        _httpConfiguration.Dispose();
-                    }
-
-                    if (_secretManager != null)
-                    {
-                        (_secretManager as IDisposable).Dispose();
-                    }
+                    _httpConfiguration.Dispose();
```

Comment:
No, not intentional... good catch. The intention was to refactor this to `_httpConfiguration?.Dispose();`